jupyterlab/labbox_ephys_widgets_jp/src/extensions/extensionInterface.ts

```diff
@@ -91,16 +152,7 @@ export interface SortingViewProps {
     selectedUnitIds: {[key: string]: boolean}
     focusedUnitId: number | null
     onUnitClicked: (unitId: number, event: {ctrlKey?: boolean, shiftKey?: boolean}) => void
-    onAddUnitLabel: (a: {
-        sortingId: string;
-        unitId: number;
-        label: string;
-    }) => void
-    onRemoveUnitLabel: (a: {
-        sortingId: string;
-        unitId: number;
-        label: string;
-    }) => void
+    curationDispatch: (action: SortingCurationAction) => void
```

Comment:
Replace the old onAddUnitLabel, onRemoveUnitLabel with the new curationDispatch. You can see that it is way better to lump together all the curation actions into one dispatch.

Reply:
Yeah, this is definitely an improvement.